PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,17 +1,15 @@
-Fixes #(issue).
+Fixes #(issue)
 
 # Done is when (DoD):
-- [ ] I have added UI/Unit tests for my changes
-- [ ] Old tests and eslint rule are not broken
-- [ ] I added modified component properties to the typescript typings
-- [ ] Design has been reviewed with (here name of reviewer)
+- [ ] My changes ship with tests, if needed.
+- [ ] I added modified component properties to the typescript typings, if needed.
+- [ ] Design changes have been reviewed with a designer.
 - [ ] My changes are accordingly to our [Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)
-- [ ] My changes are well documented in the README and showcased in the stories 
-- [ ] My changes are shortly added to CHANGELOG.md with the issue number
-- [ ] Tested with IE
+- [ ] My changes are well documented and showcased in the stories.
+- [ ] My changes are shortly added to CHANGELOG.md with the issue number.
+- [ ] Tested with IE, if affected.
```

Comment:
always needed in my opinon

Reply:
Not if you make changes on the build level.

Reply:
True 👍